src/lib/viewers/BaseViewer.js

```diff
@@ -545,8 +546,12 @@ class BaseViewer extends EventEmitter {
      */
     handleFullscreenEnter() {
         this.resize();
-        if (this.annotator) {
+
+        if (this.annotator && this.options.showAnontationsControls && this.annotationControls) {
             this.annotator.emit(ANNOTATOR_EVENT.setVisibility, false);
+
+            this.annotator.toggleAnnotationMode(AnnotationMode.NONE);
```

Comment:
Does passing nothing here work?

Reply:
> Does passing nothing here work?

No, it won't work because it sets state to `undefined` instead of `'none'`;